packages/storefront-lwc/lwc-services.config.js

```diff
@@ -1,21 +1,21 @@
 // Find the full example of all available configuration options at
 // https://github.com/muenzpraeger/create-lwc-app/blob/master/packages/lwc-services/example/lwc-services.config.js
 module.exports = {
-  buildDir: "./dist/public",
-  resources: [],
-  sourceDir: "./src",
-  moduleDir: "./src/modules",
-  server: {
-    customConfig: "./scripts/sample-app.js"
-  },
-  devServer: {
-    proxy: {
-      "/api": "http://localhost:3002",
-      "/assets": "http://localhost:3002",
-      "/manifest.json": "http://localhost:3002",
-      "/apiconfig.js": "http://localhost:3002"
+    buildDir: './dist/public',
+    resources: [],
+    sourceDir: './src',
+    moduleDir: './src/modules',
+    server: {
+        customConfig: './server/sample-app.js',
```

Comment:
does it matter if sample-app.js does not have .mjs explicitly?

Reply:
this should be change to sample-app.mjs